### DISCUSSION TITLE
argocd external redis connection to aws elastic cache cluster

### DISCUSSION THREAD

**Discussion by raghavendraDBA** [argoproj/argo-cd#16531] [Q&A]
we moved our argocd cache layer to aws elastic cache cluster for redis post that change we are noticing error related to timeout in repo server 
time="2023-12-05T10:37:20Z" level=warning msg="manifest cache set error &ApplicationSource{RepoURL:git@github.com:Jobvite-Inc/

seeing this message in UI  **Unable to load data: error getting cached app managed resources: i/o timeout**

**Comment by raghavendraDBA** [comment]
<img width="1220" alt="image" src="https://github.com/argoproj/argo-cd/assets/57623009/c52e8e99-be60-4e7c-97bc-d11957a2e217">

failed with this

**Comment by peter-hst** [comment]
same here
